lib/ariadne/data_util.rb

```diff
@@ -1,11 +1,12 @@
 
 module DataUtil
-  def self.init_redis_cli(obj = nil)
-    @redis_cli = obj
+  def self.init_redis_cli(redis_obj: nil)
+    @redis_cli = redis_obj
   end
 
-  def self.get_data_from_redis(app_name)
-    keys = @redis_cli.keys "#{app_name}*"
+  def self.get_data_from_redis(id:, app_name:)
+    key  = id.nil? ? "#{app_name}*" : "#{app_name}:#{id}"
+    keys = @redis_cli.keys key
```

Comment:
it is not possible for the caller to know the key!
This condition is there because if no `id` is specified i.e. if the `id` is `nil` then the caller must get all records related to that app.
We can't let the caller directly pass the key.
Here we are generating a required key.